src/contexts/QuotasContextProvider.tsx

```diff
@@ -0,0 +1,116 @@
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import React from 'react';
+import { clusterTemplateQuotaGVK, CLUSTER_TEMPLATES_ROLE, roleBindingGVK } from '../constants';
+import Loader from '../helpers/Loader';
+
+import { useTranslation } from '../hooks/useTranslation';
+import { Quota, QuotaDetails, RoleBinding } from '../types';
+
+const useClusterTemplateRoleBindings = () => {
+  const [rbs, loaded, error] = useK8sWatchResource<RoleBinding[]>({
+    groupVersionKind: roleBindingGVK,
+    namespaced: true,
+    isList: true,
+  });
+  if (!loaded || error) {
+    return [[], loaded, error];
```

Comment:
this is a new array on every re-render. Why is that `if` even needed ? 15-17 can be removed IMO.

Reply:
fixed